charts/bitbucket-runner-autoscaler/values.yaml

```diff
@@ -0,0 +1,63 @@
+namespace: "bitbucket-runner-controller"
+
+cleaner:
+  enabled: true
+  image: bitbucketpipelines/runners-autoscaler
```

Comment:
Can the image and version be different from the controller? If not, I would move this section under the controller -> `controller.cleaner`

Reply:
All examples I've seen online define same image same tag for both controllers. It makes sense to have them consolidated into one.